app/views/banned_adopters/edit.html.erb

```diff
@@ -1,3 +1,7 @@
-<h1><%= @title %></h1>
+<div class="container-fluid">
+    <div class="pages ml-2">
+        <h1>Edit</h1>
 
-<%= render 'form' %>
\ No newline at end of file
+        <%= render 'form' %>
+    </div>
+</div>
```

Comment:
Missing newline at the end of the file.  You can adjust the settings in vscode so that it does this automatically.

https://www.google.com/search?q=vscode+newline+end+of+file